### DISCUSSION TITLE
Importing beatmaps from Android files.

### DISCUSSION THREAD

**Discussion by SaintNoodles** [ppy/osu#22422] [Q&A] [answered]
I have a problem when importing beatmap from Android files because not all of my folder shows up, even with "tick" Hidden files. 

Also is it possible to add Vertical mania on mobile?

**Comment by Theighlin** [comment, marked as answer]
Probably a duplicate of #11911. You should be able to import maps via the game from the phone file browser though.

**Reply by mk56-spn** [reply]
In addition as to the second question vertical mania gets asked for about once a week, please use the search function https://github.com/ppy/osu/discussions/16783
